### ISSUE TITLE
Recheck if data still needs to be loaded?

### ISSUE THREAD

**Issue by marverix** [egoist/vue-mugen-scroll#15]
Hi,

I have this case that after fetchData there is still white space that mugen could still fill in with data, but he don't want to becasue there is no scroll yet. In smaller resolution it's working fine (scroll is there from very beggining) but in higher resolution it's stops to work. I have tried to experiment with couple combinations of params but no effects. I know that one solution would be to take bigger chunk of data, but still you will be able to find screen resolution that this chunk of data is unsufficient to fill whole white space.

Any ideas? Thanks!

P.S.
Great plugin mate!

**Comment by egoist** [comment]
You can put a button inside `<mugen-scroll>` component to allow user to trigger `fetchData` manually:

```vue
<mugen-scroll :should-handle="!loading" :handler="fetchData">
  <button @click="fetchData">{{ loading ? 'Loading...' : 'Load more' }}</button>
</mugen-scroll>
```

**Comment by marverix** [comment]
Hmm I don't really feel like this is a good solution... I'm looking for some kind of automated solution here...

**Comment by Sobient** [comment]
@egoist , @marverix : Mugen-scroll only shows what is handed to it. If this passes the visible scroll barrier, it will hook an event listener to do something once that point is reached. If the data is above the screen height however, there is nothing more to scroll until you send more data. 
Suggestions include: 
- Send more initial data to ensure you have enough data to cover visible screen estate in height axis
- Do your own JS calculations for height and call your `fetchData` function based upon the result

In short; Mugen-scroll does nothing wrong, and expects the developer to know (or research) what caveats exist when connecting data retrieval to scroll-linked events.

**Comment by marverix** [comment]
Ok, thanks :)
